mender/templates/deployments-deploy.yaml

```
@@ -87,6 +87,8 @@ spec:
           value: {{ .Values.deployments.env.DEPLOYMENTS_AWS_TAG_ARTIFACT | quote }}
         - name: DEPLOYMENTS_PRESIGN_SECRET
           value: {{ .Values.deployments.env.DEPLOYMENTS_PRESIGN_SECRET | quote }}
+        - name: DEPLOYMENTS_AWS_MAX_IMAGE_SIZE
+          value: {{ .Values.deployments.env.DEPLOYMENTS_AWS_MAX_IMAGE_SIZE | quote }}
```

```suggestion
          value: {{ .Values.deployments.env.DEPLOYMENTS_STORAGE_MAX_IMAGE_SIZE | default 10737418240 | int | toString | quote }}
```
A default values is missing. Either set it here, or in the `mender/values.yaml`